### DISCUSSION TITLE
how to add a timezone to event

### DISCUSSION THREAD

**Discussion by sandeep-adireddi** [ics-py/ics-py#286] [Q&A] [answered]
how to add a timezone to event

**Comment by N-Coder** [comment, marked as answer]
I'm having a really hard time trying to deduce from those 7 words what you are trying to do.
Could you maybe explain a little more what your use case is? 
It somehow feels a little disrespectful if you expect someone to take their free time to answer here if you yourself didn't event take the time to formulate a proper question.
Have you tried using a `datetime` or `arrow` with a certain timezone as begin/end time for your event?
